### ISSUE TITLE
Make the blocklist link clickable area smaller

### ISSUE THREAD

**Issue by Brilliand** [sunadarake/Personal_Blocklist#14]
With this extension active, the clickable area of the "block site" link extends all the way across the column, creating a large area where a wayward click will unexpectedly block the site.

Please change it so only the area covered by the text is clickable, like most links work.

**Comment by danowar2k** [comment]
The pull request looks fine to me (but I have only looked at it fleetingly). I'd really appreciate for it to be merged, you don't know how often I clicked at the "extended click area" to the right of the link only to go back and undo the block again.

If anyone doesn't like the smaller area, you could always provide an option later for the size of the click area ;-)

**Comment by ChocoChopin** [comment]
Yes, for the love of god, approve the pull request, I've misclicked because of this problem so many times and it's infuriating.
